src/mavsdk_server/src/mavsdk_server.cpp

```diff
@@ -35,19 +41,21 @@ class MavsdkServer::Impl {
 
         if (_server != nullptr) {
             _server->stop();
+            _server.reset();
         }
+        _mavsdk.reset();
```

Comment:
Isn't that similar to calling [`destroy`](https://github.com/mavlink/MAVSDK/blob/main/src/mavsdk_server/src/mavsdk_server_api.cpp#L53)?

I realise that MAVSDK should probably be stopped on `stop()`, but why doing it on `start()`? :thinking:

Reply:
This is in `stop()`, it's just GitHub swallowing a few lines.

Reply:
Oh, my bad